spark/wordcount/src/test/scala/com/example/bigtable/spark/wordcount/SparkSpec.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
If this is written by Google, it should have the same license header as the other file, not explicitly licensed to the ASF, but just licensed under Apache 2.0.

`WordCountIT.scala` seems to have the correct header.